Create a new sub-chapter on Embedding DS & AI across disciplines (#3852)
the-turing-way/the-turing-way: book/website/communication/open/embedding-ds-ai-disciplines.md

* **Structural Challenges**: Lack of existing frameworks and supporting structures for cross-disciplinary learning leaves educators more independent in their approach.

But also…

* **Connection Opportunities**: Linking elements such as domain knowledge, methods, people, and backgrounds that would otherwise remain unconnected can bring new, valuable insights. 
* **Multi-approach Opportunities**: Applying various disciplinary perspectives to specific case studies can enrich the outcomes.
* **Problem/solving Opportunities**: Using multiple approaches to knowledge in problem-solving discussions can yield more innovative solutions.
* **Knowledge-sharing Opportunities**: Cross-disciplinary learning equips students with skills and modes of thinking informed by multiple worldviews.
* **Flexibility**: The lack of strict frameworks can be an opportunity to explore different collaborative models and integrate existing frameworks in new and productive ways.

### Understanding the fundamental concepts of AI and Data Science 

> When teaching non-cognate students, it is crucial to first assess their understanding of AI and data science fundamentals. The multi-stage framework proposed by Kandlhofer et al. (2016) for **AI literacy** (Figure 2) can serve as a valuable reference for evaluating students' knowledge.

Reviewer: ```suggestion
When teaching non-cognate students, it is crucial to first assess their understanding of AI and data science fundamentals. 
The multi-stage framework proposed by Kandlhofer et al. (2016) for **AI literacy** (Figure 2) can serve as a valuable reference for evaluating students' knowledge.
```

* Developing a fundamental understanding of AI topics

![Screenshot (30)](https://hackmd.io/_uploads/S1CmqdRYR.png)
*Figure 2. Kandlhofer et al. (2016), Topcis of AI Literacy.*

<div class="alert alert-block alert-info">
<b>Case study: UCL, Built Environment: Sustainable Heritage MSc, Data Science route </b> 
    
This Master’s degree creates expert data scientists taught through the exciting multidisciplinary lens of cultural heritage (historic buildings, sites, landscapes, museums and collections). Students will develop advanced data science skills, such as coding, crowd-sourced data science, machine learning and data visualisation, and apply them to he complexities of acquisition, analysis and exploitation of the variety of data that is generated and used in heritage contexts.

The course is open to applicants with a technical background such as statistics or data science, as well as **applicants from other disciplines (e.g., conservation, curation, history) that want to develop data science skills**.This degree route is suited both to recent graduates and early or mid-career professionals looking to retrain or up-skill.
    

## Suggestions for Educators

> > DS educators teaching to non-cognate students would need to pay particular attention to the students' background knowledge, concepts, and practical and metacognitive skills. Assessment for leanring, differentiated instruction, collaborative learning, and other effective teaching methods, can be tailored to the unique needs of data science and AI education across disciplines.

Reviewer: ```suggestion
> DS educators teaching students without a data science or AI background would need to pay particular attention to the students' background knowledge, concepts, and practical and metacognitive skills. 
Assessment for learning, differentiated instruction, collaborative learning, and other effective teaching methods, can be tailored to the unique needs of data science and AI education across disciplines.
```

> As previously mentioned, non-cognate does not necessarily mean non-computing, non-mathematics, or non-STEM, so it is important not to make assumptions about students’ prior knowledge and skills. The suggestions below can apply to broad data science and AI education, but educators teaching non-cognate students may find them particularly useful for tailoring teaching and learning to students' needs and skillset. These different pedagogical approaches are designed to help you gain insights into your students’ understanding of specific concepts or topics, allowing you to better support their individual progress. 

Reviewer: ```suggestion
It is important not to make assumptions about students’ prior knowledge and skills. 
The suggestions below can apply to broad data science and AI education, but educators teaching students without a data science and AI background may find them particularly useful for tailoring teaching and learning to students' needs and skill sets. 
These different pedagogical approaches are designed to help you gain insights into your students’ understanding of specific concepts or topics, allowing you to better support their individual progress. 
```

> 1. **Assessment for Learning (AfL)** is used during learning, and it is useufl to identify students' demographic, students’ needs and starting point, and to generate feedback they can use to improve performance. AfL can take different forms: it could be as simple as observing class discussions, asking questions in oral or written form, or using collaborative tools such as Miro or Notion to leverage visual aids and conceptual mapping. 

Reviewer: ```suggestion
1. **Assessment for Learning** is used during learning, and it is useful to identify student demographics, student needs and starting points, and to generate feedback they can use to improve performance. 
Assessment for Learning can take different forms: it could be as simple as observing class discussions, asking questions in oral or written form, or using collaborative tools such as Miro or Notion to leverage visual aids and conceptual mapping. 
```

> AfL informs changes you can make to your lesson straight away to make it more effective. Through AfL, students will:

Reviewer: ```suggestion
Assessment for learning informs changes you can make to your lesson straight away to make it more effective. Through assessment for learning, students will:
```

* **Introduce themselves**: understanding *WHO*. General information will help you understand the overall cohort's demographics and map out discipline-specific interests.
* **Explain what they can do**: understanding *HOW*. Assess existing skills and expertise (this is also helpful for students to understand their capabilities).
* **Express preferences about what they want to do**: understanding *WHAT*. Let students describe their ideal scenarios; this will help you refine and tailor your content.
* **Share their goals and aspirations**: understanding *WHY*. What do they need this for?

> In *Seven Myths of Education* (2014), Daisy Christodoulou suggests that teachers should act as "thermostats, not thermometers" meaning they should not only measure where a student is but also make necessary adjustments to guide them to where they need to be. This perspective is fundamental when thinking about AfL, and to understand the critical role of Effective Feedback.

Reviewer: Reference to add to bib!

```suggestion
In *Seven Myths of Education* (2014), Daisy Christodoulou suggests that teachers should act as "thermostats, not thermometers" meaning they should not only measure where a student is but also make necessary adjustments to guide them to where they need to be. 
This perspective is fundamental when thinking about assessment for learning, and to understand the critical role of effective feedback.
```

Reviewer: Reference to add to bib!

> **Effective feedback in AfL**: Effective feedback requires active listening from both the educator and the student. As part of the questioning process, it is an essential tool for developing students' thinking. Feedback must be task-focused, timely, specific, clear and unbiased. In this way, you will provide your students with information about their current performance and guidance on how they can improve to reach their goals.

Reviewer: ```suggestion
**Effective feedback in assessment for learning**: Effective feedback requires active listening from both the educator and the student. 
As part of the questioning process, it is an essential tool for developing students' thinking. 
Feedback must be task-focused, timely, specific, clear and unbiased. 
In this way, you will provide your students with information about their current performance and guidance on how they can improve to reach their goals.
```

> 2. **Formative Assessment (FA)** supports teaching by assessing a learner's state and inferring next steps (Zhai et al., 2021). It is similar to AfL, as both methods are used to understand student progress and inform teaching. However, while AfL is carried out during learning to inform teaching and identify areas for improvement, FA is used for day-to-day assessments to gauge and explore students' understanding of a topic.

Reviewer: Reference to add to bib!

> 2. **Formative Assessment (FA)** supports teaching by assessing a learner's state and inferring next steps (Zhai et al., 2021). It is similar to AfL, as both methods are used to understand student progress and inform teaching. However, while AfL is carried out during learning to inform teaching and identify areas for improvement, FA is used for day-to-day assessments to gauge and explore students' understanding of a topic.

Reviewer: ```suggestion
2. **Formative Assessment** supports teaching by assessing a learner's state and inferring next steps (Zhai et al., 2021). 
It is similar to AfL, as both methods are used to understand student progress and inform teaching. 
However, while AfL is carried out during learning to inform teaching and identify areas for improvement, formative assessment is used for day-to-day assessments to gauge and explore students' understanding of a topic.
```

> The FA process usually consists of the following three practices (Stanja et al., 2023):

Reviewer: Reference to add to bib!

> The FA process usually consists of the following three practices (Stanja et al., 2023):

Reviewer: ```suggestion
The formative assessment process usually consists of the following three practices (Stanja et al., 2023):
```

Reviewer: I would write these abbreviations out in full to make it easier for people to follow

* **Eliciting**: The collection of evidence for students' learning using tasks and questions 
* **Interpreting**: Analysing what students are saying, writing or doing and what this indicates about their thinking; and identifying implications for learning based on the previous analysis
* **Responding**: Giving feedback to students or adaptation of instruction. 

Formative assessments in data science and AI education should focus on providing timely and actionable feedback that helps students improve their understanding and skills progressively. Examples include:
* **Code reviews**: Regularly assessing students’ code for functionality, efficiency, and style.
* **Concept checks**: Quick, informal assessments during lessons, such as quizzes or hands-on tasks, to gauge understanding of recent topics like machine learning algorithms or statistical methods.
* **Data challenges**: Mini group competitions where students predict outcomes based on given datasets, which are then discussed in class to learn from various approaches. When teaching in multidisciplinary settings, the challenges should be situated in the disciplinary domain that is most familiar to students.

> 3. **Differentiated Instruction (DI)** — a method that considers students’ individual learning styles and levels of readiness before designing a lesson plan (Tomlinson, 2017). 

Reviewer: ```suggestion
3. **Differentiated Instruction** — a method that considers students’ individual learning styles and levels of readiness before designing a lesson plan (Tomlinson, 2017). 
```

Reviewer: Reference to add to bib!

> DI sits between "single-size" instruction and individualised instruction, involving proactive planning of various ways for students to express their learning. While it may require fine-tuning for individual learners, offering multiple options increases the likelihood of effectively meeting the needs of many students.

Reviewer: ```suggestion
Differentiated instruction sits between "single-size" instruction and individualised instruction, involving proactive planning of various ways for students to express their learning. 
While it may require fine-tuning for individual learners, offering multiple options increases the likelihood of effectively meeting the needs of many students.
```

In this model, the teacher is viewed as an organiser of knowledge rather than a gatekeeper.

> DI in the context of data science and AI for non-cognate students by applied through:

Reviewer: ```suggestion
Differentiated instruction in the context of data science and AI can by applied through:
```

4. **Collaborative Learning**: Leveraging collaborative learning strategies can enhance understanding of data science and AI and innovation across disciplines. Techniques include:
* **Group work and Study groups**: Encourage formation of study groups that meet regularly to discuss course content and collaborate on group projects.
* **Paired programming**: Working collaboratively in pairs at a single computer helps students to plough through certain types of coding problems, supporting peer learning and knowledge sharing.
* **Peer teaching**: Similarly, assign students to teach certain concepts or technologies to their peers, reinforcing their own understanding and aiding others.
* **Flipped classrooms**: Providing students with information before class means that  students can learn at their own pace, take responsibility for their learning and actively  engage in class. This will also improve collaboration between students and between students and educators, who can get to know them better and provide better support.

## Summary

> In conclusion, teaching non-cognate students can be challenging for educators, but it also provides many opportunities, especially in terms of creativity, knowledge sharing, and problem-solving. We recommend conducting an initial assessment of students' understanding of AI and data science fundamentals (data literacy) to determine where the focus should be. It is important not to take students' existing knowledge and specific expertise for granted. Teaching approaches such as assessment for learning can be useful for this purpose, while formative assessment and differentiated instruction, combined with a blend of methods to cater to different learning styles, can support the design of teaching content and the adjustment of material according to students' needs. Collaborative learning is also an effective way to engage students, leverage their existing knowledge, and foster a supportive environment for rich exchanges and co-development.

Reviewer: ```suggestion
In conclusion, teaching students without a data science or AI background can be challenging for educators, but it also provides many opportunities, especially in terms of creativity, knowledge sharing, and problem-solving. 
We recommend conducting an initial assessment of students' understanding of AI and data science fundamentals (data literacy) to determine where the focus should be. 
It is important not to take students' existing knowledge and specific expertise for granted. 
Teaching approaches such as assessment for learning can be useful for this purpose, while formative assessment and differentiated instruction, combined with a blend of methods to cater to different learning styles, can support the design of teaching content and the adjustment of material according to students' needs. 
Collaborative learning is also an effective way to engage students, leverage their existing knowledge, and foster a supportive environment for rich exchanges and co-development.
```

---

> ### References

Reviewer: These can be added to the bibliography file :)

(cm-embedding-ds-ai-disciplines)=
# Embedding Data Science & AI across Disciplines: Context and Suggestions for Educators

> 

Reviewer: Add a note section here - you can use markdown notation to set up an "info" section

![DS](https://hackmd.io/_uploads/Syk1aiJcR.png)
*Figure 1. The interdisciplinary nature of Data Science.*

In the constantly growing data-intensive society, data science is being applied within various **non-cognate disciplines** such as arts, history, and social sciences. 
It's important for people involved in training people in these disciplines to understand how to adapt tools and develop skills in different contexts, particularly data literacy, and how educators can support the development of these specific competencies.
Data literacy is traditionally defined as the ability to explore, understand, and communicate data as information. 
This definition can be expanded by a recent contribution from Gebre (2022), who identifies key elements of data literacy, including general competencies such as attitudes toward data and specific skills like using particular tools. 
Gebre also highlights context-specific factors that impact how learners relate to data, which are highly relevant when teaching to non-cognate students.

> ### Defining Non-cognate students

Reviewer: Lets put this up in the note section
